website/documentation/getting-started/shoots/_index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Gardener Shoots
+weight: 4
+---
+
+## Overview
+
+![](./images/k8s-cluster.png)
+
+A Kubernetes cluster consists of a control plane and a data plane. The data plane runs the actual containers on nodes (which translate to physical or virtual machines). For the control and data plane to work together properly, lots of components need matching configuration.
+
+Some configurations are standardized but some are also very specific to the needs of a cluster's user / workload. Ideally, you want a properly configured cluster with the possibility to fine-tune some settings.
+
+## Concept of a "Shoot"
+
+A shoot is just another Kubernetes resource. Gardener components watch it and act upon changes (e.g., creation). It comes with reasonable default settings but also allows fine-tuned configuration. And on top of it, you get a status providing health information, information about ongoing operations, and so on.
+
+Luckily there is a dashboard to get started.
+
+## Basic Configuration Options
+
+![](./images/basic_configurations_1.png)
+
+Every cluster needs a name - after all, it is a Kubernetes resource and therefore unique within a namespace.
+
+The Kubernetes version will be used as a starting point. Once a newer version is available, you can update (but not downgrade, not supported by Kubernetes in general).
+
+The "purpose" defines some configuration (like a monitoring stack or alerting rules) and generally indicates the importance of a cluster.
+
+![](./images/basic_configurations_2.png)
+
+Start by selecting the infrastructure you want to use. The choice will be mapped to a cloud profile that contains provider specific information like the available (actual) OS images, zones and regions or machine types.
+
+Each data plane runs in an infrastructure account owned by the end user. By selecting the infrastructure secret, the user grants Gardener access to the respective account to create / manage resources.
```

Comment:
```suggestion
Each data plane runs in an infrastructure account owned by the end user. By selecting the infrastructure secret containing the accounts credentials, you are granting Gardener access to the respective account to create / manage resources.
```

The paragraph above addresses the user directly ("Start by selecting...") but indirectly here ("the user..."). Should be consistent.